index.bs

```diff
@@ -591,8 +591,14 @@ dictionary CookieStoreDeleteOptions {
   USVString path = "/";
 };
 
-dictionary CookieListItem : CookieInit {
-  boolean secure = true;
+dictionary CookieListItem {
+  required USVString name;
```

Comment:
Since this is an output-only dictionary, `required` doesn't have any meaning. Remove it?

Reply:
Removed